backend/pkg/modules/internal/bfla/bfladetector/learn_and_detect_bfla.go

```diff
@@ -88,9 +88,9 @@ type BFLADetector interface {
 	ApproveTrace(path, method string, clientRef *k8straceannotator.K8sObjectRef, apiID uint, user *DetectedUser)
 	DenyTrace(path, method string, clientRef *k8straceannotator.K8sObjectRef, apiID uint, user *DetectedUser)
 
-	ResetLearning(apiID uint, numberOfTraces int)
-	StartLearning(apiID uint, numberOfTraces int)
-	StopLearning(apiID uint)
+	ResetLearning(apiID uint, numberOfTraces int) error
+	StartLearning(apiID uint, numberOfTraces int) error
+	StopLearning(apiID uint) error
```

Comment:
Not sure why these appear as differences, in the devel-global-specs I already see them with the "error" return.

https://github.com/openclarity/apiclarity/blob/devel-global-specs/backend/pkg/modules/internal/bfla/bfladetector/learn_and_detect_bfla.go